src/yunohost/app.py

```diff
@@ -2574,6 +2576,31 @@ def unstable_apps():
     return output
 
 
+def _check_services_status_for_app(services):
+
+    logger.debug("Checking that required services are up and running...")
+
+    # Some apps use php-fpm or php5-fpm which is now php7.0-fpm
+    def replace_alias(service):
+        if service in ["php-fpm", "php5-fpm"]:
+            return "php7.0-fpm"
```

Comment:
Not directly linked to this PR, but is it a good idea to hardcode the PHP version. We know that at every debian release the php version change, so maybe could we think about a universal solution about that ?

Reply:
Meh yea but I can't really think of any simple universal solution ... But it's not like if grepping occurences of "php5" or "php7" and replacing those was the hard part :/